### PR TITLE
Adding validation for mnemonics

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Adding validations for mnemonics
### Changes

Added a validateMnemonic function
### Related Issues

#169

### CODE REVIEW

File: monitoring-bot/internal/monitor.go [hunk 1]
```diff
@@ -93,10 +94,35 @@ func NewMonitor(envPath string, jsonPath string) (Monitor, error) {
...
 
 	mon.mnemonics = map[network]string{}
-	mon.mnemonics[devNetwork] = mon.env.devMnemonic
-	mon.mnemonics[testNetwork] = mon.env.testMnemonic
-	mon.mnemonics[qaNetwork] = mon.env.qaMnemonic
-	mon.mnemonics[mainNetwork] = mon.env.mainMnemonic
+	err = validateMnemonic(mon.env.devMnemonic, "devNetwork")
+	if err != nil {
+		return mon, err
+	} else {
```
Comment (xmonader) [inline monitoring-bot/internal/monitor.go]: remove the else, if we are done with the if in 98-100 we are guaranteed there's no error
```
err = validate...
if err != nil {
  return mon, err
}
mon.mnemonics[devNetwork] = ...
```

File: monitoring-bot/internal/monitor.go [hunk 2]
```diff
@@ -109,6 +135,19 @@ func NewMonitor(envPath string, jsonPath string) (Monitor, error) {
 
 	return mon, nil
 }
+func validateMnemonic(mnemonic string, network string) error {
```
Comment (xmonader) [inline monitoring-bot/internal/monitor.go]: while this removes out the issue of having symbols, it doesn't enforce any rules on the number of words. https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/grid-client/deployer/tf_plugin_client.go#L116

File: monitoring-bot/internal/monitor_test.go [hunk 3]
```diff
@@ -37,10 +37,10 @@ func TestMonitor(t *testing.T) {
 	defer envFile.Close()
 	defer os.Remove(envFile.Name())
 
-	data = []byte(`TESTNET_MNEMONIC=mnemonic
-	MAINNET_MNEMONIC=mnemonic
-	DEVNET_MNEMONIC=mnemonic
-	QANET_MNEMONIC=mnemonic
+	data = []byte(`TESTNET_MNEMONIC=crouch kangaroo bubble riot fetch spoil chest wisdom inquiry dry toe hawk
```
Comment (ashraffouda) [inline monitoring-bot/internal/monitor_test.go]: all ppl will use ur mnemonics kda :)

**Review by ashraffouda — APPROVED** [review]

File: monitoring-bot/internal/monitor_test.go (hunk 3 above)
Comment (ashraffouda) [reply monitoring-bot/internal/monitor_test.go]: it is better if u removed it

**Review by ashraffouda — APPROVED** [review]

**Review by ashraffouda — APPROVED** [review]

**Pull request merged**